src/lib/constants.ts

```diff
@@ -69,6 +69,15 @@ const constants = {
 	// (this number is used as an upper bound when generating
 	// a random jitter)
 	maxApiJitterDelay: 60 * 1000,
+	validRedsocksProxyTypes: ['socks4', 'socks5', 'http-connect', 'http-relay'],
```

Comment:
Not sure if there is an advantage to having these constants here. They are very specific to host config, so it might make sense to put them inside the module

Reply:
Also ping @20k-ultra, this is related to my comment above on the API route

Reply:
@pipex While I do agree that these constants might be minor compared to some of the other ones in `constants.ts`, as a developer I (and potentially others) would feel more comfortable if I could reference and change constants from a single source of truth, instead of a line at the top of a module. 

In my opinion, it feels a bit more organized this way. There's also certainly no disadvantage to having constants stored in a single location that I'm aware of, unless I don't have the context for it. Thoughts?

Reply:
I agree that both can work, having it all contained has the advantage IMO that you don't have to go looking for the variable values elsewhere, and it makes it easy to split up the module if you eventually want to. 

Personally I think of constants.ts as a place to store values that will be used across the codebase, if the value is only used by a single module then i think it makes more sense to have the values in the module.

Reply:
I understand your point -- that `constants.ts` should be for more widely used constants. However, I *have* noticed multiple other constants within `constants.ts` that are only used in 1 module, plus the tests for that module, similar to how often `validRedsocksProxyTypes` is used. Maybe it's the importance or influence of a module that determines whether its constants should go in `constants.ts`?

If we're just talking about `validV1PatchProxyFields`, the other constant added, then I'd agree about removing it. Having thought about the broader scope of SV, and the fact that v1 is up for deprecation, adding this field in constants doesn't really make sense, and overall v1 is legacy & therefore not as important for the future, I suppose. However, I'm inclined to want to keep validRedsocksProxyTypes where it is, if possible, since it will also be used in the upcoming v2 endpoints (see: [v2 feature parity](https://github.com/balena-io/balena-supervisor/issues/1586)). 

I don't mean to belabor the point, of course. Eager to hear your thoughts on this compromise :)

Reply:
Yeah, I agree there is currently not much consistency in what goes into `constants.ts` right now, going forward I think it will be good if we all get to a common set of good practices we'd like to see on the supervisor codebase.

I don't want to extend the discussion too much either for this though, I think the compromise you propose is good.

Reply:
For sure! I've updated the commit.